services/cron/tasks_actions.go

```diff
@@ -18,6 +18,17 @@ func initActionsTasks() {
 	registerStopZombieTasks()
 	registerStopEndlessTasks()
 	registerCancelAbandonedJobs()
+	registerScheduleTasks()
+}
+
+func registerScheduleTasks() {
+	RegisterTaskFatal("start_schedule_tasks", &BaseConfig{
+		Enabled:    true,
+		RunAtStart: true,
```

Comment:
I don't think we need `RunAtStart` for cronjobs.
```suggestion
		RunAtStart: false,
```